web/pdf_page_view.js

```diff
@@ -654,31 +687,49 @@ class PDFPageView {
     redrawAnnotationEditorLayer = false,
     redrawXfaLayer = false,
     redrawTextLayer = false,
+    postponeDrawingDelay = -1,
```

Comment:
It doesn't seem that this parameter is still used in the latest version of the patch.